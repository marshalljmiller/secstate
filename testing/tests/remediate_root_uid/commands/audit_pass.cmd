/usr/bin/secstate audit rootuid-req
