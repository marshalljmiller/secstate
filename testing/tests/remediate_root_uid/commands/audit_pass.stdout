--Results for 'rootuid-req' (Profile: 'fix-conflicts')--
Passed:		1
Mitigated:	0
Failed:		0
Fixed:		0
Not Selected:	2
Not Checked:	0
Not Applicable:	0
Error:		0
Informational:	0
Unknown:	0

